### DISCUSSION TITLE
Good Tutorial For Using StableSwarmUI against a rented GPU

### DISCUSSION THREAD

**Discussion by ataylorm** [Stability-AI/StableSwarmUI#206] [Q&A] [answered]
Does anyone happen to know of a good tutorial for how to use StableSwarmUI against rented GPU's?  How do you do it?  How do you handle getting models, loras, and the like uploaded, etc?

**Comment by mcmonkey4eva** [comment, marked as answer]
You can use the Colab notebook on any service that gives you notebook access https://github.com/Stability-AI/StableSwarmUI/tree/master/colab
RunPod and Vast.AI are the two most popular GPU server providers right now, iirc both give notebook setup options so that's probably the easy route.

**Reply by ataylorm** [reply]
Thanks, since I have no Linux experience or Colab experience I was hoping there might be a more complete guide somewhere.  I'll keep searching Google.
